scripts/zones/Southern_San_dOria/npcs/Kipopo.lua

```diff
@@ -29,11 +52,22 @@ function onTrigger(player,npc)
 end;
 
 function onEventUpdate(player,csid,option)
+
 end;
 
 function onEventFinish(player,csid,option)
-    if (csid == 651 and option == 1) then
+    if (csid == 912) then
+        player:giveKeyItem(dsp.ki.REPAIRED_HANDBAG)
```

Comment:
`player:giveKeyItem` doesn't exist, it's either `player:addKeyItem` (if using the [raw core function](https://github.com/DarkstarProject/darkstar/blob/master/src/map/lua/lua_baseentity.cpp#L6460-L6467)) or `npcUtil.giveKeyItem(player, dsp.ki.REPAIRED_HANDBAG)` (if using [npcUtil function](https://github.com/DarkstarProject/darkstar/blob/master/scripts/globals/npc_util.lua#L235-L244)). Core's version just gives it silently. npcUtil's version gives it _and_ displays the obtained message.